setup.py

```diff
@@ -110,6 +110,9 @@ def __init__(self, name, extra_cmake_args=[]):
 DEP_BUILD_DIR = os.path.join(PROJECT_DIR, 'build', 'deps')
 DEP_INSTALL_PATH = os.environ.get('AWS_C_INSTALL', os.path.join(DEP_BUILD_DIR, 'install'))
 
+if sys.platform != 'darwin' and sys.platform != 'win32':
+    AWS_LC_INSTALL = os.environ.get('AWS_LC_INSTALL', os.path.join(DEP_BUILD_DIR, 'libcrypto'))
```

Comment:
is `AWS_LC_INSTALL` a standard environment variable we're using in other CRTs as well?

If it's something we're introducing here for the 1st time, let's remove the acronym and name it `AWS_LIBCRYPTO_INSTALL`

Reply:
ok